forte/sparse_ci/determinant.h

```diff
@@ -41,6 +42,7 @@ size_t constexpr Norb2 = 2 * Norb;
 
 using String = BitArray<Norb>;
 using Determinant = DeterminantImpl<Norb2>;
+using Configuration = ConfigurationImpl<Norb2>;
```

Comment:
It doesn't look like `Configuration` is used in this file. Is it included here so that pybind can access it with the existing `#include "sparse_ci/determinant.h"`? If so, mark this resolved.